src/pages/test-report/cat-cpc/__tests__/test-report.cat-cpc.page.spec.ts

```diff
@@ -0,0 +1,126 @@
+import { ModuleAssessmentComponent } from '../components/module-assessment/module-assessment';
+import { QuestionAnswerComponent } from '../components/question-answer/question-answer';
+import { QuestionCardComponent } from '../components/question-card/question-card';
+import { QuestionFooterComponent } from '../components/question-footer/question-footer';
+import { QuestionScoreComponent } from '../components/question-score/question-score';
+import { QuestionSubtitleComponent } from '../components/question-subtitle/question-subtitle';
+import { QuestionTitleComponent } from '../components/question-title/question-title';
+import { catCPCTestData, mockToggleEvent } from '../__mocks__/test-report.cat-cpc.mock';
+import { QuestionNumber } from '../../../../shared/constants/cpc-questions/cpc-question-combinations.constants';
+
+describe('TestReportCatCPCPage', () => {
+  let fixture: ComponentFixture<TestReportCatCPCPage>;
+  let component: TestReportCatCPCPage;
+  // let navController: NavController;
+  let store$: Store<StoreModel>;
+
+  configureTestSuite(() => {
+    TestBed.configureTestingModule({
+      declarations: [
+        TestReportCatCPCPage,
+        MockComponent(CompetencyComponent),
+        MockComponent(ModuleAssessmentComponent),
+        MockComponent(QuestionAnswerComponent),
+        MockComponent(QuestionCardComponent),
+        MockComponent(QuestionFiveCardComponent),
+        MockComponent(QuestionFooterComponent),
+        MockComponent(QuestionScoreComponent),
+        MockComponent(QuestionSubtitleComponent),
+        MockComponent(QuestionTitleComponent),
+      ],
+      imports: [
+        IonicModule,
+        AppModule,
+        StoreModule.forFeature('tests', () => (
+          {
+            currentTest: {
+              slotId: '123',
+            },
+            testStatus: {},
+            startedTests: {
+              123: {
+                testData: initialState,
+                journalData: {
+                  candidate: candidateMock,
+                },
+              },
+            },
+          })),
+        StoreModule.forFeature('testReport', testReportReducer),
+      ],
+      providers: [
+        { provide: NavController, useFactory: () => NavControllerMock.instance() },
+        { provide: NavParams, useFactory: () => NavParamsMock.instance() },
+        { provide: Config, useFactory: () => ConfigMock.instance() },
+        { provide: Platform, useFactory: () => PlatformMock.instance() },
+        { provide: AuthenticationProvider, useClass: AuthenticationProviderMock },
+        { provide: ModalController, useFactory: () => ModalControllerMock.instance() },
+        { provide: CPCQuestionProvider, useClass: CpcQuestionsMock },
+      ],
+    });
+  });
+
+  beforeEach(async(() => {
+    fixture = TestBed.createComponent(TestReportCatCPCPage);
+    component = fixture.componentInstance;
+    // navController = TestBed.get(NavController);
```

Comment:
same here - looks like it can be removed now

Reply:
same for that, will be removed